src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -38,14 +41,15 @@
 	 * @return Returns a reference to the loaded 
 	 */
 	public static CCIgnore createForProject(IProject p) {
-		IFile file = p.getFile(new Path(Paths.get(FILENAME).normalize().toString()));
+		IFile file = p.getFile(new Path(FILENAME));
 		
 		if (!file.exists()) {
 			InputStream in = new ByteArrayInputStream(DEFAULT_CONTENTS.getBytes());
 			
 			try {
 				// warn directory watching
-				PluginManager.getInstance().putFileInWarnList(file.getProjectRelativePath().toString(), FileCreateResponse.class);
+				IPath workspaceRelativePath = p.getFullPath().append(file.getProjectRelativePath());
```

Comment:
Use file.getFullPath() instead, should return the workspace relative path for the file